### PR TITLE
feat: add full screen loader to field

### PR DESCRIPTION
**Pull request by drewradcliff**

### CODE REVIEW

File: app/components/field.tsx [hunk 1]
```diff
@@ -34,8 +34,28 @@ export function Field() {
     );
   }, [size]);
 
-  // todo: add skeleton loader
-  return !plots ? null : (
+  return !plots ? (
+    <div className="absolute bg-white h-full w-full left-0 right-0">
```
Comment (lewxdev) [inline app/components/field.tsx]: may not need `left-0` and `right-0` if you have `h-full` and `w-full` (which can be `w-[100dvw]` `h-[100dvh]`.
also, do we need z-index here?

File: app/components/field.tsx [hunk 2]
```diff
@@ -34,8 +34,28 @@ export function Field() {
...
+      <div className="h-full flex items-center justify-center">
+        <div
+          style={{
+            display: "grid",
+            gap: `${GAP_SIZE}rem`,
+            gridTemplateColumns: `repeat(3, ${GRID_SIZE}rem)`,
+          }}
+          className="animate-pulse"
```
Comment (lewxdev) [inline app/components/field.tsx]:
```suggestion
          className="grid animate-pulse"
          style={{
            gap: `${GAP_SIZE}rem`,
            gridTemplateColumns: `repeat(3, ${GRID_SIZE}rem)`,
          }}
```

File: app/components/field.tsx [hunk 3]
```diff
@@ -34,8 +34,28 @@ export function Field() {
...
+        >
+          {Array.from({ length: 9 }).map((_, index) => (
+            <div
+              key={index}
+              className="bg-gray-100"
+              style={{ height: `${GRID_SIZE}rem`, width: `${GRID_SIZE}rem` }}
+            />
+          ))}
```
Comment (lewxdev) [inline app/components/field.tsx]:
```suggestion
          {Array.from({ length: 9 }, (_, index) => (
            <div
              key={index}
              className="bg-gray-100"
              style={{ height: `${GRID_SIZE}rem`, width: `${GRID_SIZE}rem` }}
            />
          ))}
```

File: app/components/header.tsx
Comment (lewxdev) [inline app/components/header.tsx:1]: haha, whoopsie

**Review by lewxdev — CHANGES_REQUESTED** [review]

File: app/components/header.tsx [hunk 4]
```diff
@@ -16,18 +16,24 @@ export function Header() {
...
+                <span>{clientsCount}</span>
+                <UsersIcon className="h-4 w-4" />
+              </>
+            )}
           </div>
-        )}
+          <div className="flex items-center justify-end gap-2">
+            {!!exposedPercent && (
+              <>
+                <span className="">{exposedPercent}</span>
```
Comment (drewradcliff) [inline app/components/header.tsx]:
```suggestion
                <span>{exposedPercent}</span>
```

File: app/components/header.tsx [hunk 5]
```diff
@@ -16,18 +16,24 @@ export function Header() {
...
-              <Percent className="h-4 w-4" />
-            </div>
+        <div className="flex flex-col items-end gap-2">
+          <div className="flex items-center gap-2">
+            {!!clientsCount && (
+              <>
+                <span>{clientsCount}</span>
+                <UsersIcon className="h-4 w-4" />
+              </>
+            )}
```
Comment (lewxdev) [inline app/components/header.tsx]:
```suggestion 
          {!!clientsCount && (
            <div className="flex items-center gap-2">
              <span>{clientsCount}</span>
              <UsersIcon className="h-4 w-4" />
            </div>
          )}
```

File: app/components/header.tsx [hunk 6]
```diff
@@ -16,18 +16,24 @@ export function Header() {
...
           </div>
-        )}
+          <div className="flex items-center justify-end gap-2">
+            {!!exposedPercent && (
+              <>
+                <span>{exposedPercent}</span>
+                <Percent className="h-4 w-4" />
+              </>
+            )}
+          </div>
```
Comment (lewxdev) [inline app/components/header.tsx]:
```suggestion
          {!!exposedPercent && (
            <div className="flex items-center justify-end gap-2">
              <span>{exposedPercent}</span>
              <Percent className="h-4 w-4" />
            </div>
          )}
```

**Review by lewxdev — CHANGES_REQUESTED** [review]
last one and it should be good 👍🏽

**Review by lewxdev — APPROVED** [review]
🤞🏽

**Pull request merged**